docs/application/dotnet/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,142 @@
+
+        ```
+        string componentId = compoRunningContext.ComponentId;
+        string applicationId = compoRunningContext.ApplicationId;
+        string instanceId = compoRunningContext.InstanceId;
+        ```
+
+    -   Check the state of the component:
+
+        ```
+        if (compoRunningContext.State == ComponentRunningContext.ComponentState.Initialized)
+            /// The component is constructed.
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Created)
+            /// The component is created.
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Started)
+            /// The component is started.
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Resumed)
+            /// The component is resumed.
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Paused)
+            /// The component is paused
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Destroyed)
+            /// The component is destroyed.
+        else
+            /// State is undefined
+        ```
+
+    -   Resume the running application:
+
+        ```
+        compoRunningContext.Resume();
+        ```
+
+<a name="filter"></a>
+## Getting the Component Information
+
+To get the installed information and its details, and to operate on the information:
+
+1.  Get the information of the currently-installed component by creating an instance of the [Tizen.Applications.ComponentBased.ComponentInfo](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentInfo.html) class, with the ID of the component from which the information is being obtained as a parameter.
+
+    When an component is not installed, it is impossible to get its information.
+
+    ```
+    ComponentInfo compoInfo = new ComponentInfo(Your Component ID);
+    ```
+
+2.  Operate on the context:
+    -   Get the component ID and application ID:
+
+        ```
+        string componentId = compoInfo.ComponentId;
+        string applicationId = compoInfo.ApplicationId;
+        ```
+    -   Check the type of the component:
+
+        ```
+        if (compoInfo.ComponentType == ComponentType.Frame)
+            /// The component is frame-component.
+        else if (compoInfo.ComponentType == ComponentType.Service)
+            /// The component is service-component
+        else
+            /// ComponentType is undefined
+        ```
+
+    -   Get the label and the icon path:
+
+        ```
+        string label = compoInfo.Label;
+        string iconPath = compoInfo.IconPath;
+        ```
+
+3.  Call the `GetInstalledComponentsAsync()` method of the [Tizen.Applications.ComponentBased.ComponentManager](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentManager.html) class and retrieve all components and print their information:
```

Comment:
Suggest rephrasing to:

3.  Call the `GetInstalledComponentsAsync()` method of the [Tizen.Applications.ComponentBased.ComponentManager](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentManager.html) class, and retrieve all components and print their information:

Reply:
Done